queries/landuse.jinja2

```diff
@@ -29,19 +48,27 @@ WHERE
 UNION ALL
 
 SELECT
-    name,
-    {{ ne_landuse_cols('park or protected land') }}
+  name,
+  operator,
+  way_area::bigint AS area,
+  COALESCE("boundary", "landuse", "leisure", "natural", "highway", "aeroway", "amenity", "tourism", "man_made", "power") AS kind,
```

Comment:
We've been adding the new coalesce values at the end of the list. 

Should we:

* Keep that pattern, adding "boundary" after "power
* Alpha sort them all (new and old)
* Add at the beginning of the list (as you propose)